compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -684,7 +684,9 @@ static void addLowerToLLVMPasses(OpPassManager &passManager,
 
 void buildLLVMCPUCodegenConfigurationPassPipeline(OpPassManager &passManager) {
   {
-    addCommonTargetExecutablePreprocessingPasses(passManager);
+    addCommonTargetExecutablePreprocessingPasses(
+        passManager,
+        /* useDecomposeSoftmaxFusion */ false);
```

Comment:
I think this should be `/*...=*/false`. See https://google.github.io/styleguide/cppguide.html#Function_Argument_Comments

```suggestion
        /*useDecomposeSoftmaxFusion=*/false);
```